### ISSUE TITLE
dis-aggregate events

### ISSUE THREAD

**Issue by kra** [futel/usage#27]
Can't show graphs with two lines for events
for example in the old viz we used to be able to see "operator" and "operator-nopickup" independently, but now we only see one line for the totals

https://futel.github.io/usage/?p=680%7C640%7C515%7C510%7C520%7C610%7C700%7C690%7C695%7C550%7C660%7C668%7C645%7C670%7C615%7C505%7C620%7C655%7C625%7C630&e=operator%7Coperator-nopickup&a=weekly&r=-2m&c=1#

**Comment by kra** [comment]
choose to aggregate or not aggregate as with the phones currently

**Comment by breedx2** [comment]
Yeah, this was definitely not in the new design (events are always combined). If you want this, it shouldn't be _too_ bad really.....
Do you want the entire permutation space of N phones x M events, or does separate series for events imply combining phones?

**Comment by kra** [comment]
I think the events and phones should both have a "combine items" check similar to how phones has currently. It should have the entire permutation but if it's easier if one must be combined that's fine, keep it simple, I presume that when there are too many permutations the graph quickly becomes useless anyway.
